static/js/public/store/filters.js

```diff
@@ -152,39 +160,21 @@ class Filters {
     }
   }
 
-  isFilterElement(el) {
-    return el.dataset.js && el.dataset.js === "filter";
-  }
-
-  initFilterEvents(el) {
-    el.addEventListener("click", (e) => {
-      let target = e.target;
-
-      while (!this.isFilterElement(target) || !target.parentNode) {
-        target = target.parentNode;
-      }
-
-      if (target) {
-        e.preventDefault();
-        const removeEl = target.querySelector("[data-js='remove-filter']");
-
-        const filterType = target.dataset.filterType;
-        const filterValue = target.dataset.filterValue;
-
-        if (this.filterExists(filterType, filterValue)) {
-          this.removeFilter(filterType, filterValue);
-          target.classList.remove("is-active");
-          removeEl.classList.add("u-hide");
-        } else {
-          this.addFilter(filterType, filterValue);
-          target.classList.add("is-active");
-          removeEl.classList.remove("u-hide");
-        }
-
-        this.cleanFilters();
-        this.updateHistory();
+  syncSortUI(selectorName, newValue) {
```

Comment:
This makes me uncomfortable, but let's wait until @danielmutis has had a think about the UX some more and see if we need to remove/ update this later.

Reply:
agree. I'm sure we can come up with a better solution once Daniel has the design/UX for it